app/assets/scss/_search_overview.scss

```diff
@@ -18,7 +18,7 @@ ol.instruction-list-item-top li {
 
 ol.steps {
```

Comment:
Why do we have this scss file here in the buyer app? Shouldn't it be in the frontend toolkit and imported?

Reply:
I think because we required additional styling. @samuelhwilliams could you confirm?

Reply:
If it's possible to get it in the frontend toolkit, that'd be great. I don't remember why it was put here specifically. It might even just have been my lack of understanding at the time, or I was in a rush. Not ideal.

If it's easy to re-centralise this, we should.